src/battle_script_commands.c

```diff
@@ -14901,7 +14905,7 @@ static void Cmd_handleballthrow(void)
                 break;
             case ITEM_LURE_BALL:
                 if (gIsFishingEncounter)
-                    ballMultiplier = (B_LURE_BALL_MODIFIER >= GEN_7 ? 500 : 300);
+                    ballMultiplier = (B_LURE_BALL_MODIFIER >= GEN_7 ? ((B_LURE_BALL_MODIFIER) >= GEN_8 ? 400: 500) : 300);
```

Comment:
This just looks ugly.
```suggestion
                {
                    if (B_LURE_BALL_MODIFIER >= GEN_8)
                        ballMultiplier = 400;
                    else if (B_LURE_BALL_MODIFIER >= GEN_7)
                        ballMultiplier = 500;
                    else
                        ballMultiplier = 300;
                }
```

Reply:
I don't agree, but I *would* say that newbies would have a harder time understanding it.

Reply:
We should decide whether we allow multi-ternaries or not.

Personally I don't mind them for configs but I dislike them in anything else.

Reply:
Afaiac, ternaries are fine for things like if/else, but when there's a 3rd statement involved it starts to get more and more convoluted the deeper you go into that rabbit hole, therefore it becomes less and less readable for newbies and eventually it just becomes too hard to interpret at a glance.